tests/test_ui_tabs_enhanced.py

```
@@ -114,7 +114,7 @@ async def test_jobs_tab_contains_table():
         try:
             jobs_table = pilot.app.query_one("#jobs-table-full", DataTable)
             assert jobs_table is not None
-        except NoMatches:
+        except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)